docs/changelog/0.7.rst

```diff
@@ -19,11 +19,12 @@ Release description not yet written.
 New features
 ------------
 
-* Enhancements in message forwarding
+* Added support for live locations
```

Comment:
Please add your changelog _after_ the one is in the master branch

Reply:
Sorry, ctrl+v in the wrong spot!